From: Tomas Henzl <thenzl@redhat.com>
Date: Fri, 18 Dec 2009 23:23:25 -0500
Subject: [fusion] remove unnecessary printk
Message-id: <20091218192303.2799.25290.sendpatchset@localhost.localdomain>
Patchwork-id: 22206
O-Subject: [RHEL6 1/3] bz548408 fusion: remove unnecessary printk
Bugzilla: 548408
RH-Acked-by: David Milburn <dmilburn@redhat.com>

remove unnecessary printk

Signed-off-by: Aristeu Rozanski <arozansk@redhat.com>

diff --git a/drivers/message/fusion/mptctl.c b/drivers/message/fusion/mptctl.c
index 9b2e219..352acd0 100644
--- a/drivers/message/fusion/mptctl.c
+++ b/drivers/message/fusion/mptctl.c
@@ -621,11 +621,8 @@ __mptctl_ioctl(struct file *file, unsigned int cmd, unsigned long arg)
 	 */
 	iocnumX = khdr.iocnum & 0xFF;
 	if (((iocnum = mpt_verify_adapter(iocnumX, &iocp)) < 0) ||
-	    (iocp == NULL)) {
-		printk(KERN_DEBUG MYNAM "%s::mptctl_ioctl() @%d - ioc%d not found!\n",
-				__FILE__, __LINE__, iocnumX);
+	    (iocp == NULL))
 		return -ENODEV;
-	}
 
 	if (!iocp->active) {
 		printk(KERN_DEBUG MYNAM "%s::mptctl_ioctl() @%d - Controller disabled.\n",
